src/Driver/driver_configs.jl

```diff
@@ -167,16 +171,17 @@ function AtmosLESConfiguration(
     numerical_flux_first_order = RusanovNumericalFlux(),
     numerical_flux_second_order = CentralNumericalFluxSecondOrder(),
     numerical_flux_gradient = CentralNumericalFluxGradient(),
+    fv_reconstruction = nothing,
 ) where {FT <: AbstractFloat}
 
     (polyorder_horz, polyorder_vert) = get_polyorders(N)
 
     print_model_info(model)
 
     brickrange = (
-        grid1d(xmin, xmax, elemsize = Δx * polyorder_horz),
-        grid1d(ymin, ymax, elemsize = Δy * polyorder_horz),
-        grid1d(zmin, zmax, elemsize = Δz * polyorder_vert),
+        grid1d(xmin, xmax, elemsize = Δx * max(polyorder_horz, 1)),
+        grid1d(ymin, ymax, elemsize = Δy * max(polyorder_horz, 1)),
+        grid1d(zmin, zmax, elemsize = Δz * max(polyorder_vert, 1)),
```

Comment:
Should we apply these changes to the other configurations?

Reply:
update the land one